Tests/UnitTests/Core/Utilities/GridAxisGeneratorsTests.cpp

```diff
@@ -45,7 +45,7 @@ BOOST_AUTO_TEST_CASE(Eq1D) {
   BOOST_CHECK(axisC.getBoundaryType() == AxisBoundaryType::Closed);
 
   // Test that we can make a grid out of this
-  EqBound::grid_type<bool> eqbGrid(std::move(axisTupleB));
+  EqBound::grid_type<std::byte> eqbGrid(std::move(axisTupleB));
```

Comment:
Haha, of course I had used it ... anyhow, the `GridAxisGenerator` will also vanish.